styles.css

```diff
@@ -274,3 +282,93 @@ body {
     border-radius: 8px;
     background-color: #363636;
 }
+
+/* Switch styles */
+.switch {
+    position: relative;
+    display: inline-block;
+    width: 34px;
+    height: 20px;
+}
+
+.switch input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+}
+
+.slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #ccc;
+    transition: .4s;
+    border-radius: 20px;
+}
+
+.slider:before {
+    position: absolute;
+    content: "";
+    height: 12px;
+    width: 12px;
+    left: 4px;
+    bottom: 4px;
+    background-color: white;
+    transition: .4s;
+    border-radius: 50%;
+}
+
+input:checked + .slider {
+    background-color: #124875;
+}
+
+input:checked + .slider:before {
+    transform: translateX(14px);
+}
+
+label {
+    display: block;
+    margin-top: 20px;
+}
+
+label.switch {
+    display: inline-block;
+    margin-left: 10px;
+}
+
+#apply-settings {
+    background-color: #444;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    cursor: pointer;
+    margin-top: 20px;
+    border-radius: 15px;
+}
+
+#apply-settings:hover {
+    background-color: #555;
+}
+
+/**/
+
+.settings-inline {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    margin-top: 0px; /* Space between each setting */
+}
+
+.settings-inline label {
+    margin-right: 30px; /* Space between label and input */
+    white-space: nowrap;
+}
+
+.settings-inline select {
+    margin-right: 5px; /* Additional space for better alignment */
```

Comment:
For alignment, this seems to have done the trick:
```suggestion
    margin: 20px auto 0px 5px; /* Additional space for better alignment */
```
![image](https://github.com/Animeboynz/Mihon-Backup-Viewer/assets/7104931/672ec145-8244-4ac6-8d73-dc645077e78d)

Also, the button is visible but disabled during the file selection step